### PR TITLE
Feature/pelagos 3799 add stats

### PR DESCRIPTION
**Pull request by fightingtexasaggie**

### CODE REVIEW

File: src/Controller/UI/StatsController.php [hunk 1]
```diff
@@ -0,0 +1,299 @@
+<?php
+
+namespace App\Controller\UI;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Query;
+
+use Pelagos\Bundle\AppBundle\Twig\Extensions as TwigExtentions;
```
Comment (praneethpr) [inline src/Controller/UI/StatsController.php]: Need to change namespace for this

Comment (fightingtexasaggie) [reply src/Controller/UI/StatsController.php]: good catch. I wonder how it's even working, or if it's even needed.

Comment (praneethpr) [reply src/Controller/UI/StatsController.php]: If you goto the /stats/json route, it breaks and that is used in the drupal home page

File: webpack.config.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ Encore
     // will require an extra script tag for runtime.js
     // but, you probably want this, unless you're building a single-page app
     //.enableSingleRuntimeChunk()
-
```
Comment (mickel1138) [inline webpack.config.js]: ?

Comment (fightingtexasaggie) [reply webpack.config.js]: soft tab got removed. My editor cleans this stuff up automatically.

**Pull request merged**